src/main/kotlin/command/Timer.kt

```diff
@@ -0,0 +1,84 @@
+package command
+
+import chat.ChatUtility
+import chat.Formatting
+import io.papermc.paper.command.brigadier.CommandSourceStack
+import net.kyori.adventure.bossbar.BossBar
+import net.kyori.adventure.text.Component
+import org.bukkit.Bukkit
+import org.bukkit.entity.Player
+import org.bukkit.scheduler.BukkitRunnable
+import org.incendo.cloud.annotations.Argument
+import org.incendo.cloud.annotations.Command
+import org.incendo.cloud.annotations.Default
+import org.incendo.cloud.annotations.Permission
+import org.incendo.cloud.annotations.processing.CommandContainer
+import plugin
+import util.Sounds.PLING
+import util.timeRemainingFormatted
+
+@Suppress("unused", "unstableApiUsage")
+@CommandContainer
+class Timer {
+    @Command("timer start <time> [countdown]")
+    @Permission("tbd.command.timer")
+    fun timerStart(css: CommandSourceStack, @Argument("time") time: Int, @Argument("countdown") @Default("-1") countdown: Int) {
+        if(time in 1..60) {
```

Comment:
I am fairly certain cloud has the ability to do range restriction on the argument itself. This would then also provide a nice error when a user is typing the wrong value range.